modules/tinymce/src/core/main/ts/api/Formatter.ts

```diff
@@ -39,7 +39,7 @@ interface Formatter extends FormatRegistry {
   match: (name: string, vars?: FormatVars, node?: Node) => boolean;
   closest: (names) => string | null;
   matchAll: (names: string[], vars?: FormatVars) => string[];
-  matchNode: (node: Node, name: string, vars?: FormatVars, similar?: boolean) => boolean;
+  matchNode: (node: Node, name: string, vars?: FormatVars, similar?: boolean) => Format | undefined;
```

Comment:
Please include a changelog entry for this fix

- [ ] Add a fixed changelog entry

Reply:
Also added.